### PR TITLE
Add Wyre

### PR DESCRIPTION
**Pull request by afritz**
I struggled to choose the folder bc the co name is "Wyre" but the URL is "sendwyre". I opted to use the s directory and base on url rather than corpname.

### CODE REVIEW

File: img/s/sendwyre.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...3 1.577-.301.837-.488 1.723-.67 2.598-.457 2.19-.88 4.387-1.327 6.617m13.75-10.859c.615-.747 1.722-1.16 2.544-.903 1.003.314 1.332 1.425.62 2.201a2.45 2.45 0 0 1-.915.608c-1.035.388-2.106.376-3.232.182.19-.826.492-1.491.983-2.088zm5.505 7.318h-.144c-1.647.631-3.474.797-4.734.44-1.391-.395-2.194-1.606-1.941-2.974 4.147.35 5.908-.44 6.853-1.005 2.56-1.531 3.21-5.63-.13-6.843-2.935-1.066-7.023-.559-8.954 1.781-1.76 2.132-2.277 4.477-2.007 7.087.202 1.963 1.549 4.778 5.949 5.217 2.66.266 4.363-.418 4.486-.469l.622-3.234z"/><path fill="#94C5FF" d="M16.474 15.927h9.78l-5.276 12.888a4.75 4.75 0 0 1-4.397 2.95h-9.78l5.276-12.887a4.75 4.75 0 0 1 4.397-2.951"/><path fill="#1A32AF" d="m26.254 15.927-5.922 7.53c-.897 1.196-2.256 1.973-3.853 1.973H9.395l2.69-6.556a4.75 4.75 0 0 1 4.395-2.947h9.774Z"/><path fill="#0092FF" d="M13.232 8.005h9.78l-5.277 12.889a4.75 4.75 0 0 1-4.396 2.95H3.56l5.276-12.888a4.75 4.75 0 0 1 4.397-2.95"/><path fill="#1A32AF" d="m23.012 8.006-5.922 7.53c-.897 1.196-2.256 1.973-3.853 1.973H6.153l2.69-6.556a4.75 4.75 0 0 1 4.395-2.947h9.774Z"/><path fill="#05F" d="M9.99.076h9.78l-5.277 12.889a4.75 4.75 0 0 1-4.396 2.95H.317L5.593 3.027A4.75 4.75 0 0 1 9.99.077"/></g></svg>
```
Comment (Carlgo11) [inline img/s/sendwyre.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100"><path d="M35.4 19.8h30.3v8L63 34h11v8l-3.3 8h10v6.3l-18.5 24H27.4v-7l3-7H19.2V59z"/><path fill="#fff" d="M71 37H45.4l-15 36.3h4.3l18-23.4h14.8l3.5-8.3ZM37.1 22.4 22.3 59h4.1l17.2-24.7H60l3-7v-4.8z"/></svg>
```

File: entries/s/sendwyre.com.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "Wyre": {
+    "domain": "sendwyre.com",
+    "additional-domains": [
+      "dash.sendwyre.com"
+    ],
+    "tfa": [
+      "sms",
+      "custom-software"
+    ],
+    "documentation": "https://support.sendwyre.com/hc/en-us/articles/360057063573-Enabling-your-Multi-Factor-Authenticator",
```
Comment (Carlgo11) [inline entries/s/sendwyre.com.json]:
```suggestion
    "documentation": "https://support.sendwyre.com/hc/en-us/articles/360057063573",
```

File: entries/s/sendwyre.com.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "recovery": "https://support.sendwyre.com/hc/en-us/articles/360055319134-I-can-t-access-my-account-with-MFA",
```
Comment (Carlgo11) [inline entries/s/sendwyre.com.json]:
```suggestion
    "recovery": "https://support.sendwyre.com/hc/en-us/articles/360055319134",
```

File: entries/s/sendwyre.com.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "Wyre": {
+    "domain": "sendwyre.com",
+    "additional-domains": [
+      "dash.sendwyre.com"
+    ],
```
Comment (Carlgo11) [inline entries/s/sendwyre.com.json]: The domain covers all subdomains so this part isn't needed.
```suggestion
```

File: entries/s/sendwyre.com.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    "tfa": [
+      "sms",
+      "custom-software"
```
Comment (Carlgo11) [inline entries/s/sendwyre.com.json]: Looks like TOTP is used.
```suggestion
      "totp"
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by afritz** [comment]
reviewed and committed changes, tx. made the commits in this UI. If thats wrong pls lmk - I don't do many public PRs. 

Looks like there 403 is still returned from the documentation and recovery elements values. Checked both URLs with httpstatus.io -> sendwyre.com is returning 403 Forbidden on these links, in both versions, e.g., with the added text at the end and the recommended change of removing the text. The links work fine in a browser, curl etc. Should they be removed from the json or can the 403 be ignored?

**Comment by Carlgo11** [comment]
The pull request looks fine. The 403 warnings are just warnings and not errors so it can be ignored. Wyre probably uses some kind of proxy which blocks our [bots](https://2fa.directory/bot/) but as you mentioned, the pages display fine in web browsers.

**Review by Carlgo11 — APPROVED** [review]

File: img/s/sendwyre.com.svg [hunk 6]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100"><path d="M35.4 19.8h30.3v8L63 34h11v8l-3.3 8h10v6.3l-18.5 24H27.4v-7l3-7H19.2V59z"/><path fill="#fff" d="M71 37H45.4l-15 36.3h4.3l18-23.4h14.8l3.5-8.3ZM37.1 22.4 22.3 59h4.1l17.2-24.7H60l3-7v-4.8z"/></svg>
```
Comment (phallobst) [inline img/s/sendwyre.com.svg]: Better leave out the empty padding:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 61.5 60.5"><path d="M16.2 0h30.3v8l-2.7 6.2h11v8l-3.3 8h10v6.3L43 60.5H8.2v-7l3-7H0v-7.3z"/><path fill="#fff" d="M51.8 17.2H26.2l-15 36.3h4.3l18-23.4h14.8l3.5-8.3zM17.9 2.6 3.1 39.2h4.1l17.2-24.7h16.4l3-7V2.7Z"/></svg>
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**